.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Bug Report Checklist
+
+- [ ] I have checked the existing [documentation](https://github.com/finbourne/${REPO_NAME}/wiki)
+- [ ] I have searched for any related issues/PRs
+- [ ] I have read the [LUISD contribution guidelines](https://github.com/finbourne/${REPO_NAME}/blob/master/docs/CONTRIBUTING.md)
+- [ ] I have read the [LUSID Code of Conduct](https://github.com/finbourne/${REPO_NAME}/blob/master/docs/CODE_OF_CONDUCT.md)
+
+**Version**
+The version you are using that has the bug. 
+
+**IMPORTANT: DO NOT POST ANY SENSITIVE DATA E.G. PASSWORDS**
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
might be good to gather platform as well.  windows/linux/mac, + version of such things.

Reply:
done